Pipfile

```diff
@@ -65,10 +63,13 @@ pytest-cov = "*"
 pytest-datadir-ng = "*"
 pytest-mock = "*"
 pytest-subprocess = "*"
+pytest-freezegun = "*"
 requests-mock = "*"
 tox-pyenv = "*"
 Deprecated = "*"
 pipfile-requirements = "*"
+matplotlib = "*"
```

Comment:
We prefer to avoid dependencies, please alternative ways to do the plotting (if it's necessary).
Message me for more information

Reply:
removed